com.unity.render-pipelines.universal/Runtime/Passes/CopyDepthPass.cs

```diff
@@ -136,6 +136,7 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
                 {
                     // Blit has logic to flip projection matrix when rendering to render texture.
                     // Currently the y-flip is handled in CopyDepthPass.hlsl by checking _ProjectionParams.x
+                    // However special exception is done by Overlay camera which doesn't have _ProjectionParams.x set up
```

Comment:
The overlay camera should setup the correct params though. I think the reason this might not be set is because this is set on SetupCameraProperties which today is only called for base camera. I'd argument that at some point we remove the dependency for SetupCameraProperties and setup all camera properties consistently for both base and overlay cameras. cc @lukaschod for some thoughts as I think you last touched the SetupCameraProperties.
 https://sourcegraph.com/github.com/Unity-Technologies/Graphics/-/blob/com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs?L682-693

Reply:
_ProjectionParams should be setup on overlay, @jonuuukas could u check maybe it is wrong by chance. As I had to recreate logic in C# which is not 100% accurate to c++ side

Reply:
Yeah, i was wondering whether we should call SetupCameraProperties for Overlay camera, but took the easy way for the time being. I could change this if you'd like

Reply:
so i fixed this in 8e9f70829cbcb3cfa65eb50962240f79b1b0d115 - previously the check in SetPerCameraShaderVariables wasn't 100% match with what we do in c++ (IsCameraProjectionMatrixFlipped matches though). Nothing much changes though - instead of only checking whether targetTexture != null, we'll also check if it's rendering to not-backbuffer (which requires flip)